data/json/effects.json

```diff
@@ -1811,15 +1811,17 @@
   {
     "type": "effect_type",
     "id": "hypocalcemia",
-    "name": [ "Hypocalcemia", "Weak bones", "Brittle bones" ],
-    "desc": [ "A lack of calcium in your diet will make your bones progressively weaker." ],
-    "apply_message": "Your bones are becoming more brittle.",
-    "remove_message": "Your bones regain their usual strength.",
+    "name": [ "Mild Hypocalcemia", "Hypocalcemia", "Acute Hypocalcemia" ],
+    "desc": [ "A lack of calcium in your diet will cause various neuromuscular symptoms." ],
+    "apply_message": "Your muscles are becoming more stiff.",
+    "remove_message": "Stiffness in your musles passes away.",
     "decay_messages": [
       [ "Your calcium deficiency is nearly resolved.", "good" ],
-      [ "Your bones become stronger as your calcium deficiency improves.", "good" ]
+      [ "Your have better control of your muscles as your calcium deficiency improves.", "good" ]
     ],
     "max_intensity": 3,
+    "base_mods": { "dex_mod": [ -1 ], "speed_mod": [ -10 ] },
+    "scaling_mods": { "dex_mod": [ -2 ], "speed_mod": [ -15 ] },
     "rating": "bad",
     "blood_analysis_description": "Hypocalcemia"
```

Comment:
See below. Developing hypocalc*emia*, meaning lowered blood levels because of low intake just does not happen in real life, unless you have severe chronic GI problems (repeated episodes of pancreatitis or the like ), or kidney failure. Both of those are failure conditions in the postapocalypse, and otherwise tracking calcium intake would only be of consequence if bones get properly implemented.